core/services/workflows/delegate.go

```diff
@@ -37,13 +38,16 @@ func (d *Delegate) OnDeleteJob(context.Context, job.Job) error { return nil }
 
 // ServicesForSpec satisfies the job.Delegate interface.
 func (d *Delegate) ServicesForSpec(ctx context.Context, spec job.Job) ([]job.ServiceCtx, error) {
+	cma := custmsg.NewLabeler().With(wIDKey, spec.WorkflowSpec.WorkflowID, woIDKey, spec.WorkflowSpec.WorkflowOwner, wnKey, spec.WorkflowSpec.WorkflowName)
 	sdkSpec, err := spec.WorkflowSpec.SDKSpec(ctx)
 	if err != nil {
+		logCustMsg(cma, fmt.Sprintf("failed to convert to sdk workflow spec: %v", err), d.logger)
```

Comment:
Can we have the message read as:
"failed to start workflow engine: <more granular message, eg. error getting sdk spec>: %s"

For this line and every other line in this function?

This is supposed to be user-facing so we'll need more context

Reply:
sure thing, I can do that.